### PR TITLE
Add header to avoid CORB,  server/index.js line 35

### PR DESCRIPTION
**Pull request by Chris-Hesterman**

### CODE REVIEW

**Comment by Chris-Hesterman** [comment]
yes the tests fail because of unusual functionality of client/proxy code inherited

File: server/index.js [hunk 1]
```diff
@@ -32,7 +32,9 @@ app.use('*.js', function (req, res, next) {
 
 app.use(router);
 
-app.use(serveStatic(path.join(__dirname, '../client/public')));
+app.use(serveStatic(path.join(__dirname, '../client/public')), (res) => {
+  res.set('Access-Control-Allow-Origin', '*');
+});
```
Comment (samasastudio) [inline server/index.js:38]: muuuch cleaner.  may steal this setup for myself!

**Review by samasastudio — APPROVED** [review]
ship it!  nice work Chris.

File: server/index.js (hunk 1 above)
Comment (Chris-Hesterman) [reply server/index.js:38]: we shall see if it works!

**Pull request merged**